content/en/news/kubesphere-meetup-2020.md

```diff
@@ -0,0 +1,64 @@
+---
+
+title: 'Recap: KubeSphere & Friends 2020 Meetup'
+keywords: KubeSphere, Community, Meetup, Kubernetes
+description: KubeSphere and Friends 2020 Meetup attracted more than 100 individual community members and honored guests to share their ideas and experiences on cloud-native and Kubernetes.
+createTime: '2021-02-02'
+author: 'Feynman, Sherlock'
+
+---
+
+![Snip20210201_8](/images/news/meetup-2020/Snip20210201_8.png)
+
+Around the latest technology trend – 'cloud-native,' the annual event KubeSphere and Friends 2020, also known as Kubernetes and Cloud-Native Meetup, was held on December 19th, 2020 in Beijing, attracting more than one hundred Kubernetes enthusiasts and KubeSphere community members.
+
+Not only did our individual community members participate in the event, our guests from CNCF and AWS, and other enterprise users all shared their ideas and experiences from different perspectives.
+
+
+
+![Snip20210201_9](/images/news/meetup-2020/Snip20210201_9.png) 
+
+​                     *Tally Lee  from CNCF Member Alliance Manager*
+
+
+
+![Snip20210201_14](/images/news/meetup-2020/Snip20210201_14.png)
+
+​                               *Zhanling Chen from AWS China*
+
+
+
+![Snip20210201_10](/images/news/meetup-2020/Snip20210201_10.png)
+
+​                              *Ray Zhou   Head of KubeSphere*
+
+
+
+We also provided on-site workshops to discuss and try the most common use features of KubeSphere.
+
+ ![Snip20210201_20](/images/news/meetup-2020/Snip20210201_20.png)
+
+![Snip20210201_22](/images/news/meetup-2020/Snip20210201_22.png)
+
+ 
+
+KubeSphere has been recognized by many uses across the world and has developed a more diversified and vibrant community. Though so many passionate users could not present at the event in person, they generously shared their experience and expectations for KubeSphere in videos. 
```

Comment:
a more diversified and vibrant community
->
a diversified and vibrant community

Delete "more" here as we do not have any comparison here.